### DISCUSSION TITLE
Help in tuning trace sampling rate in OAP

### DISCUSSION THREAD

**Discussion by manikandantk88** [apache/skywalking#11082] [General]
Hi,

I am trying to reduce the trace sampling rate in OAP but it is not working. My trace objectives are 10% of trace samples, all erroneous request traces, and traces of requests take more than 1s. 

I went through this document and configured the trace sampling rate in my OAP server.
https://skywalking.apache.org/docs/main/v9.1.0/en/setup/backend/trace-sampling/

This is my sampling rate configuration. 
![image](https://github.com/apache/skywalking/assets/34866824/5d60df88-1f47-4fe1-9107-68cd4a2a33b1)

After this configuration, I am still seeing traces of all requests. Am I missing anything here? Please help. @wu-sheng

**Comment by wu-sheng** [comment]
I don't know, maybe this config isn't loaded correctly? Basically, that should work.

**Comment by yqjdcyy** [comment]
- use nacos to sync the config, and it works
```logs
Nacos config changed: agent-analyzer.defau
lt.traceSamplingPolicy: default:
  rate: 100
  duration: -1
```
- but it seem to be not work for trace, the metric of [remote out count] not change and cpu/ memory still

- do you have any idea about this?

**Comment by wu-sheng** [comment]
This is server side sampling, not agent side.

**Reply by yqjdcyy** [reply]
After your remind and reRead the doc, oap.sampling just works on storage of segments, so it do not works on memory and cpu
